apps/dashboard/app/models/script.rb

```diff
@@ -137,6 +137,18 @@ def save
     false
   end
 
+  def destroy!
```

Comment:
Should just be `destroy`. Methods with `!` raise errors, this seems to catch the error and return true or false.

Reply:
will update